EventFilter/RPCRawToDigi/python/RPCCPPFRawToDigi_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+RPCCPPFRawToDigi = cms.EDProducer('RPCAMCRawToDigi',
```

Comment:
capitalization rule applies to python module names as well. Instances should start with a lower case